### ISSUE TITLE
[Bug]: In High contrast aquatic theme keyboard focus is not visible on primary button

### ISSUE THREAD

**Issue by frankwang32** [microsoft/fluentui#28983]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (20) x64 Intel(R) Core(TM) i9-10900 CPU @ 2.80GHz
    Memory: 31.75 GB / 63.71 GB
  Browsers:
    Edge: Spartan (44.22621.2134.0), Chromium (115.0.1901.203)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/nifty-wilbur-wjttx3?file=/example.tsx

## Actual Behavior
under high contrast aquatic theme, keyboard focus on the primary button is not very clear.

## Expected Behavior
under high contrast aquatic theme, keyboard focus on the primary button should be clear.

### Requested priority

High

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
